### ISSUE TITLE
fort.search() raises TypeError in example.js 

### ISSUE THREAD

**Issue by rohanp** [stoffern/pokemongo-api#36]
```
TypeError: Cannot read property 'fort_id' of undefined
    at Checkpoint.search (src/Fort.js:19:18)
    at _callee$ (example/example.js:50:18)
    at tryCatch (/Users/rohanp/hack/pokemongo-api/node_modules/regenerator-runtime/runtime.js:62:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/rohanp/hack/pokemongo-api/node_modules/regenerator-runtime/runtime.js:336:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/rohanp/hack/pokemongo-api/node_modules/regenerator-runtime/runtime.js:95:21)
    at step (example/example.js:2:1)
    at example/example.js:2:1
    at run (/Users/rohanp/hack/pokemongo-api/node_modules/core-js/modules/es6.promise.js:87:22)
    at /Users/rohanp/hack/pokemongo-api/node_modules/core-js/modules/es6.promise.js:100:28
    at flush (/Users/rohanp/hack/pokemongo-api/node_modules/core-js/modules/_microtask.js:18:9)
```

**Comment by stoffern** [comment]
@rohanp try updating to last commit and try again..
npm i -S github:stoffern/pokemongo-api

**Comment by stoffern** [comment]
@rohanp it is also published in npm now v0.1.9
